### ISSUE TITLE
How to reproduce the result

### ISSUE THREAD

**Issue by drx-code** [buggyyang/CDC_compression#9]
Thank you for your great work! I tried to reproduce the result of the epsilon-param method. However, the FID score is slightly higher than the downloaded checkpoints. Here are the hyper-parameters in my training in the warm-up stage:

```
# training config
n_step = 500000
scheduler_checkpoint_step = 100000
log_checkpoint_step = 5000
gradient_accumulate_every = 1
lr = 5e-5
decay = 0.8
minf = 0.4
optimizer = "adamw"  # adamw or adam
n_workers = 4

# load
load_model = False
load_step = False

# diffusion config
pred_mode = 'noise'
loss_type = "l1"
iteration_step = 20000
sample_steps = 500
embed_dim = 64
dim_mults = (1, 2, 3, 4, 5, 6)
hyper_dim_mults = (4, 4, 4)
context_channels = 3
clip_noise = "none"
val_num_of_batch = 1
additional_note = ""
vbr = False
context_dim_mults = (1, 2, 3, 4)
sample_mode = "ddim"
var_schedule = "linear"
aux_loss_type = "lpips"
compressor = "big"

# data config
data_config = {
    "dataset_name": "vimeo",
    "data_path": "/home/Vimeo90K",
    "sequence_length": 1,
    "img_size": 256,
    "img_channel": 3,
    "add_noise": False,
    "img_hz_flip": False,
}

batch_size = 4

result_root = "result"
tensorboard_root = "result"
```

In the training stage, I revised the hyper-parameters as 

```
# training config
n_step = 1500000
scheduler_checkpoint_step = 600000
log_checkpoint_step = 5000
gradient_accumulate_every = 1
lr = 5e-5
decay = 0.8
minf = 0.4
optimizer = "adamw"  # adamw or adam
n_workers = 4

# load
load_model = True
load_step = True

# diffusion config
pred_mode = 'noise'
loss_type = "l1"
iteration_step = 20000
sample_steps = 500
embed_dim = 64
dim_mults = (1, 2, 3, 4, 5, 6)
hyper_dim_mults = (4, 4, 4)
context_channels = 3
clip_noise = "none"
val_num_of_batch = 1
additional_note = ""
vbr = False
context_dim_mults = (1, 2, 3, 4)
sample_mode = "ddim"
var_schedule = "linear"
aux_loss_type = "lpips"
compressor = "big"

# data config
data_config = {
    "dataset_name": "vimeo",
    "data_path": "/home/Vimeo90K",
    "sequence_length": 1,
    "img_size": 256,
    "img_channel": 3,
    "add_noise": False,
    "img_hz_flip": False,
}

batch_size = 4

result_root = "result"
tensorboard_root = "result"
```

I ran the code with `python train.py --device 0 --beta 0.00001 --alpha 0.9` in the warm-up stage and  `python train.py --device 0 --beta 0.2048 --alpha 0.9` in the training stage.  I want to know if there is an issue with my hyperparameters or training strategy settings. Thank you!

**Comment by buggyyang** [comment]
try to use adam instead of adamw? I remembered results may probably deterioate with adamw. Note that at the warm-up stage, we simply use alpha=0. All the ckpts (with different betas/alphas) were fine-tuned from the same warmed-up model. Optionally you can try to use beta=1e-6 and slightly more warm-up steps (like 700,000) to guarantee that the model is sufficiently warmed-up. Hope it help

**Comment by drx-code** [comment]
OK, I'll try it. Thank you!
